recipes/oommfc/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "oommfc" %}
+{% set version = "0.7.6" %}
+{% set sha256 = "f54761c3519cd255b51cddaeed7c90235e1912075ea335e2de7a2b436634e36c" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  skip: True  # [py27]
+  skip: True  # [win32]
+  number: 0
+  script: pip install --no-deps .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - sarge
+    - scipy
+    - micromagneticmodel
+    - oommfodt
+    - oommf
+    - pytest
```

Comment:
`pytest` should be in `test: requires:` as it is not a run dependency

Reply:
Thanks, but in this case it is actually a run time requirement: there is a function `oommfc.test()` that can be run after importing the module which will use `purest` to run the test suite of the package.

Reply:
I think we had a similar discussion on one of our other packages before. It's debatable whether that counts as a runtime requirement or not, but we've left it as such in the discretisedfield package, for example:

https://github.com/conda-forge/discretisedfield-feedstock/blob/0733df2ee365de503c678523a900497180a4f077/recipe/meta.yaml#L30

Reply:
Okay